### PR TITLE
Add documentation of heroku deployment

### LINKED ISSUE #56: Motivating Documentation
**I'm submitting a ...**  (check one with "x")
- [X] feature request

**Current behavior:**

Currently, we have a deployment to heroku.

**Expected behavior:**

To make it motivating for others to contribute, we can tell them how long it takes for their changes to become live.

- [ ] Document how the automatic deployment works, where to find the deployed version
- [ ] Document what to do to get your changes live
- [ ] Document, how long it takes after merge to see your changes inline.

### PR DESCRIPTION
**Pull request by S2606**
Fixes #56 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
Please add the installation instructions under the installation heading.
A rough outline:
Step 0: Install python 3
Step 1: `pip install -r app/requirmnts.txt`
Step 2: `sudo apt-get install librsvg2-bin pdftk`
Step 3: `python app/main.py`

**Comment by meets2tarun** [comment]
> Please add the installation instructions under the installation heading.

This will goes under local deployment section. Make the section so that one can use the anchor link to that section for redirecting.
An example to help you out:
https://github.com/fossasia/meilix-generator#installation

@S2606

**Comment by S2606** [comment]
@meets2tarun Thank you and I am working on it

**Comment by S2606** [comment]
@meets2tarun @jajodiaraghav  Please review

**Comment by meets2tarun** [comment]
For reviewing the changes please provide the link to your forked repo of that branch deployed on Heroku.
@S2606

**Comment by S2606** [comment]
github.com/S2606/badgeyay/tree/readme-changes

**Comment by niccokunzmann** [comment]
@S2606 This looks good. If you like, you can create 3 first-timer-issues for people to go through the different setup steps and they should report and improve everything they wish different. This way, we may get three new contributors ^^

**Comment by niccokunzmann** [comment]
@meets2tarun What do you think?

**Comment by meets2tarun** [comment]
> github.com/S2606/badgeyay/tree/readme-changes

This is not what I asked for. Please deploy your readme-changes branch on Heroku and provide the link for testing.
@S2606

**Comment by meets2tarun** [comment]
> What do you think?

Yes @niccokunzmann That's a good idea to create `first-timer-issues`.
@S2606 Go for it! 👍

**Comment by niccokunzmann** [comment]
@meets2tarun You would need to pass a potitive review in order to merge this. I do not understand what you mean by deploying the markdown files.

**Comment by meets2tarun** [comment]
@niccokunzmann 
Ok, Pardon me, I was making mistake since I got myself confuse with the two PR's.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,21 @@ This is an Open Source project and we would be happy to see contributors who rep
...
+
+2. [Deployment on Heroku](/docs/installation/heroku.md)
+
+3. [Deployment with Docker](/docs/installation/docker.md)
+
+One-click Heroku deployment is also available:
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+
```
Comment (meets2tarun) [inline README.md]: We can here also include a tile for Travis Pass and fail test.

**Comment by meets2tarun** [comment]
@S2606 Thanks!

**Pull request merged**